src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests.csproj

```diff
@@ -36,6 +36,12 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
If a subfolder is added for expected JSON output, the added code is no longer needed.